src/index.js

```diff
@@ -6,7 +6,10 @@ let L = 'en' // global locale
 const Ls = {} // global loaded locale
 Ls[L] = en
 
-const isDayjs = d => d instanceof Dayjs // eslint-disable-line no-use-before-define
+const IS_DAYJS = '$isDayjsObject'
+
+// eslint-disable-next-line no-use-before-define
+const isDayjs = d => d instanceof Dayjs || !!(d && d[IS_DAYJS])
```

Comment:
I have a problem related to this change. We have a Dayjs object that we store in `location.state` where it get's serialized. Upon retrieving said object we use `isDayjs` to check whether the object is actually of type `Dayjs` or if we have to parse it, something along the lines of

```ts
const maybeBrokenDayjs(obj: any) => {
  if (isDayjs(obj) {
    return obj;
  } else if (obj?.$d) {
    return dayjs(obj.$d)
  }
}
```

This now obviously breaks as `isDayjs` will suddenly yield `true`. While this is of course easily fixable it is a breaking change and thus quite unexpected in a Patch-Release :(